torchvision/ops/poolers.py

```diff
@@ -2,7 +2,7 @@
 from torch import nn, Tensor
 
 import torchvision
-from torchvision.ops import roi_align
+from .roi_align import roi_align
```

Comment:
just curious, was this flagged somehow?

Reply:
See https://github.com/pytorch/vision/pull/4178#discussion_r669382906. In short: we relied on a specific order of the imports due to a naming conflict. Without this fix we need to exclude these imports from being sorted.

Reply:
There might have been a reason for that initially, but I'm wondering why we even use absolute imports within the `torchvision` package. Every torchvision improt should be relative there. Perhaps @fmassa knows?

(anyway, this is for another PR, let's leave it as is :)  )